groovy/postInit/chemistry/ChemistryOverhaul.groovy

```diff
@@ -3580,18 +3599,30 @@ LCR.recipeBuilder()
         .EUt(30)
         .buildAndRegister()
 
+LCR.recipeBuilder()
+        .fluidInputs(fluid('ethylenediamine') * 1000)
+        .fluidInputs(fluid('formaldehyde') * 4000)
+        .inputs(ore('dustSodiumCyanide') * 12)
+        .fluidInputs(fluid('water') * 4000)
+        .fluidInputs(fluid('hydrochloric_acid') * 4000)
+        .fluidOutputs(fluid('ammonia') * 4000)
+        .fluidOutputs(fluid('ethylenediaminetetraacetic_acid_solution') * 4000)
+        .duration(300)
+        .EUt(30)
+        .buildAndRegister()
+
 BR.recipeBuilder()
         .fluidInputs(fluid('ethylenediaminetetraacetic_acid_solution') * 1000)
         .inputs(ore('dustSodiumHydroxide') * 3)
```

Comment:
needs to be 12 NaOH

Reply:
4B of H4EDTA solution contains 1 mol of H4EDTA, and 1 mol of H4EDTA reacts with 12 NaOH dust
-> 1B of H4EDTA solution needs 3 NaOH dust

Reply:
ignore as this recipe will be deleted anyways